v2/spanner-common/src/main/java/com/google/cloud/teleport/v2/spanner/migrations/avro/AvroToValueMapper.java

```diff
@@ -192,6 +195,23 @@ static Long avroFieldToLong(Object recordValue, Schema fieldSchema) {
     }
   }
 
+  static Float avroFieldToFloat(Object recordValue, Schema fieldSchema) {
```

Comment:
nit: maybe rename the method to float32 and the below method to float64? Its more clear that way.

Reply:
Fixed. Thanks . Float is usually float32 unless specified, but it follows spanner convention to call out 32.